pinot-spi/src/main/java/org/apache/pinot/spi/config/table/assignment/InstanceAssignmentConfig.java

```diff
@@ -28,6 +28,8 @@
 
 public class InstanceAssignmentConfig extends BaseJsonConfig {
 
+  @JsonPropertyDescription("Configuration for the strategy to assign instances to partitions")
+  public final PartitionSelector _partitionSelector;
```

Comment:
Make it private since you've already had a getter method?

Reply:
added